docs/book/features/volume_expansion.md

```diff
@@ -1,120 +1,78 @@
 
-Create a PVC with a StorageClass that allows volume expansion.
-
-```bash
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: example-vanilla-block-pvc
-spec:
-  accessModes:
-  - ReadWriteOnce
-  resources:
-    requests:
-      storage: 1Gi
-  storageClassName: example-vanilla-block-sc
-```
-
-```bash
-kubectl apply -f example-pvc.yaml
-```
-
-Increase the size of the PVC. Note that PVC cannot be attached to a Pod as only offline volume expansion is supported.
+Prior to increasing the size of a PVC make sure that the PVC is bound and is not attached to a Pod as only offline volume expansion is supported.
 
-```bash
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: example-vanilla-block-pvc
-spec:
-  accessModes:
-  - ReadWriteOnce
-  resources:
-    requests:
-      storage: 2Gi
-  storageClassName: example-vanilla-block-sc
-```
-
-When the PVC is not used by any pod, modify the PVC configuration to trigger volume expansion.
+Patch the PVC to increase its size:
 
 ```bash
-kubectl apply -f example-pvc.yaml
+kubectl patch pvc example-block-pvc -p '{"spec": {"resources": {"requests": {"storage": "2Gi"}}}}'
 ```
 
-The Cloud Native Volume size in vCenter will be changed to the new size. Size of PV will also be changed with that.
-
-Note: The size of PVC won’t be changed until the PVC is used by a new Pod.
+This will trigger an expansion in the volume associated with the PVC in vSphere Cloud Native Storage which finally gets reflected on the size of the corresponding PV object. Note that the size of PVC will not change until the PVC is attached to a node i.e used by a Pod.
 
 ```bash
 kubectl get pv
-NAME                                       CAPACITY ACCESS MODES RECLAIM POLICY STATUS   CLAIM STORAGECLASS           REASON AGE
-pvc-9e9a325d-ee1c-11e9-a223-005056ad1fc1   2Gi        RWO       Delete   Bound default/example-vanilla-block-pvc   example-vanilla-block-sc 6m44s
+NAME                                       CAPACITY ACCESS MODES RECLAIM POLICY STATUS   CLAIM                       STORAGECLASS           REASON AGE
+pvc-9e9a325d-ee1c-11e9-a223-005056ad1fc1   2Gi           RWO         Delete     Bound    default/example-block-pvc   example-block-sc              6m44s
 
 kubectl get pvc
-NAME                        STATUS VOLUME                         CAPACITY ACCESS MODES STORAGECLASS AGE
-example-vanilla-block-pvc   Bound pvc-9e9a325d-ee1c-11e9-a223-005056ad1fc1   1Gi        RWO       example-vanilla-block-sc   6m57s
+NAME                STATUS VOLUME                                     CAPACITY ACCESS MODES   STORAGECLASS       AGE
+example-block-pvc   Bound  pvc-9e9a325d-ee1c-11e9-a223-005056ad1fc1   1Gi           RWO       example-block-sc   6m57s
 ```
 
-Note: Size of the PVC is unchanged. There is also a `FilesystemResizePending` condition on the PVC.
+As you can see above, the size of the PVC is unchanged. You will also notice a `FilesystemResizePending` condition applied on the PVC when you `describe` it.
```

Comment:
they may also see a Resizing condition if they are using thick provisioning

Reply:
Could you explain this? I am not aware of this scenario

Reply:
if you expand a thick provisioned volume, depending on the backend and expanded size they may see `Resizing` status condition on the PVC for a while before seeing`FilesystemResizePending` status condition.

Reply:
Every volume expansion call will go through the `Resizing` condition first before finally reaching `FileSystemResizePending` condition after ControllerExpandVolume is successful, IMO we do not need to mention the internal details.